packages/xdl/src/Project.ts

```diff
@@ -2105,6 +2127,32 @@ export async function stopExpoServerAsync(projectRoot: string): Promise<void> {
   });
 }
 
+async function startDevServerAsync(projectRoot: string, startOptions: StartOptions) {
+  _assertValidProjectRoot(projectRoot);
+
+  const port = await _getFreePortAsync(19000); // Create packager options
+  await ProjectSettings.setPackagerInfoAsync(projectRoot, {
+    expoServerPort: port,
+    packagerPort: port,
+  });
+
+  const options: MetroDevServerOptions = {
+    port,
+    logger: ProjectUtils.getLogger(projectRoot),
+  };
+  if (startOptions.reset) {
+    options.resetCache = true;
+  }
+  if (startOptions.maxWorkers != null) {
+    options.maxWorkers = startOptions.maxWorkers;
+  }
+  if (startOptions.target) {
+    options.target = startOptions.target;
```

Comment:
Thoughts on moving `EXPO_TARGET` here to keep the dev server package more agnostic?

Reply:
Good idea 👍